tdrs-backend/tdpservice/settings/common.py

```diff
@@ -270,14 +270,17 @@ class Common(Configuration):
 
 
     # Django Rest Framework
+    DEFAULT_RENDERER_CLASSES = ['rest_framework.renderers.JSONRenderer']
+    TEST_REQUEST_RENDERER_CLASSES = ['rest_framework.renderers.JSONRenderer']
+    if DEBUG:
```

Comment:
Nice work adding debug here! :)

Reply:
thanks for this comment @raftmsohani. good to know that this is what disables access to v1/ in prod.